tools/lkl/lib/hijack/hijack.c

```diff
@@ -291,7 +304,25 @@ int select(int nfds, fd_set *r, fd_set *w, fd_set *e, struct timeval *t)
 	return lkl_call(__lkl__NR_select, 5, nfds, r, w, e, t);
 }
 
-HOOK_CALL_USE_HOST_BEFORE_START(epoll_create);
+HOST_CALL(epoll_create);
+int epoll_create(int flags)
```

Comment:
Hi Hajime

I submitted a new pull request with HOST_CALL to the https://github.com/lkl/linux. My assumption is that DPDK needs epoll_create. Could you please help or provide a bit more feedback?

Thank you in advance,
  Octavian S